src/common/config/config.hpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2025 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_COMMON_CONFIG_CONFIG_HPP_
+#define AOS_COMMON_CONFIG_CONFIG_HPP_
+
+#include <string>
+#include <vector>
+
+#include <aos/common/monitoring/resourcemonitor.hpp>
+#include <aos/common/tools/error.hpp>
+
+#include <common/utils/json.hpp>
+
+namespace aos::common::config {
+
+/***********************************************************************************************************************
+ * Types
+ **********************************************************************************************************************/
+
+/*
+ * Journal alerts configuration.
+ */
+struct JournalAlerts {
+    std::vector<std::string> mFilter;
+    int                      mServiceAlertPriority;
+    int                      mSystemAlertPriority;
+};
+
+/*
+ * Migration configuration.
+ */
+struct Migration {
+    std::string mMigrationPath;
+    std::string mMergedMigrationPath;
+};
+
+/***********************************************************************************************************************
+ * Functions
+ **********************************************************************************************************************/
+
+/*
```

Comment:
I expect here only one method:

```c++
/*
 * Parses config from file.
 *
 * @param filename config file name.
 * @param[out] config config instance.
 * @return Error.
 */
Error ParseConfig(const std::string& filename, Config& config);
```

Similar to SM.

Reply:
But this is a general function in common, which is used in SM and CM